components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.throttling.synchronizer/src/main/java/org/wso2/carbon/apimgt/micro/gateway/throttling/synchronizer/dto/ConditionalGroupDTO.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2017 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we make it 2018? There are several other places we need to fix.

Reply:
These dto classes were taken from the carbon apimgt and therefore we didn't do any modifications to them. I will correct the year in the license header.